spec/models/sequence_spec.rb

```diff
@@ -171,80 +127,12 @@
       }
     end
 
-    it 'returns a hash for a simple sequence that can be consumed by the Portal' do
-      expect(sequence.serialize_for_portal('http://test.host')).to eq(simple_portal_hash)
-    end
-
-    it 'returns a hash for a sequence with activities that can be consumed by the Portal' do
-      expect(sequence_with_activities.serialize_for_portal("http://test.host")).to eq(complex_portal_hash)
-    end
-
     it 'returns a simple hash for an Activity Player sequence that can be consumed by the Portal' do
       expect(activity_player_sequence.serialize_for_portal('http://test.host')).to eq(ap_simple_portal_hash)
     end
 
   end
 
-  describe '#serialize_for_report_service' do
```

Comment:
Doesn't this still exist? LARA still needs to send the activity structure to the report service in the report-service format.

Reply:
Yes, added back (and updated to add preview_url) in https://github.com/concord-consortium/lara/pull/1074/commits/4d0c77235530e1302f0d8d54f3c239569437ddb6